config/software/openssl.rb

```diff
@@ -27,12 +27,16 @@
 
 # OpenSSL source ships with broken symlinks which windows doesn't allow.
 # Skip error checking.
-source url: "https://www.openssl.org/source/openssl-#{version}.tar.gz", extract: :lax_tar
+source url: "https://www.openssl.org/source/old/#{version}/openssl-#{version}.tar.gz", extract: :lax_tar
```

Comment:
i believe there is a typo here:
```
curl -v https://www.openssl.org/source/old/1.0.2u/openssl-1.0.2u.tar.gz
...
< HTTP/1.1 404 Not Found
< Server: Apache/2.4.29 (Ubuntu)
< Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
< Accept-Ranges: bytes
< Content-Type: text/html; charset=UTF-8
< Content-Length: 4182
< Cache-Control: max-age=172800
< Expires: Wed, 08 Apr 2020 19:10:48 GMT
< Date: Mon, 06 Apr 2020 19:10:48 GMT
< Connection: keep-alive
```

Reply:
Nice catch. Sorry about that. Should be fixed now.